### ISSUE TITLE
KMS AccessDenied on list-key-policies for default keys

### ISSUE THREAD

**Issue by bacoboy** [Netflix/security_monkey#721]
You cannot list a key policy for AWS default keys (where the alias starts with `aws/`).  For instance:
```
% aws --profile security-admin kms list-key-policies --key-id XXXX-XXX-XXX-XXXXXX

An error occurred (AccessDeniedException) when calling the ListKeyPolicies operation: User: arn:aws:sts::123456789012:assumed-role/Admin/AWS-CLI-session-1494443865 is not authorized to perform: kms:ListKeyPolicies on resource: arn:aws:kms:us-east-1:123456789012:key/XXXX-XXX-XXX-XXXXXX
```

You can easily see this going to IAM->Encryption Keys and picking the key-id for any of the default keys.

SecurityMonkey shouldn't even attempt to list key policies for these keys as it creates a lot of cloudtrail noise and unnecessary API traffic.

Can we modify the scanner to skip keys with aliases that start with `aws/`?

**Comment by monkeysecurity** [comment]
That may be a good idea.  I'm working on another branch right now so a PR would be appreciated.

**Comment by bacoboy** [comment]
Not familiar with the codebase, but wanted to report the behavior.  Perhaps @AlexCline (the original contributor) would be interested in fixing up?

**Comment by monkeysecurity** [comment]
Fixed in #735. Closing this issue.

**Comment by bacoboy** [comment]
I knew Alex would come through!  Thanks @AlexCline !
